src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -53,15 +83,31 @@ internal class NodePlatform : IProgrammingPlatform
             _environment = environment;
         }
 
+        /// <summary>
+        /// Gets the name of Node.js platform which this generator will create builds for.
+        /// </summary>
         public string Name => NodeConstants.NodeJsName;
```

Comment:
We already have documentation for this field, and for several others, in the base interface - `IProgrammingPlatform`. Let's try to avoid duplicating text. If StyleCop complains about this, let's fix the StyleCop ruleset :)

Reply:
Right. There have lots of duplicating texts. I think a better way is to write documentation only for those fields that didn't appear in IProgrammingPlatform.cs and try to reset the rule for the rest :)